README.md

```diff
@@ -1,93 +1,48 @@
-# Contributing
+# TypeScript Plugin for Autorest
 
-This project welcomes contributions and suggestions. Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.microsoft.com.
+[Autorest](https://github.com/Azure/autorest/blob/master/docs/readme.md) is a suite of tools to automatically generate SDKs for cloud services. This project provides a backend that generates SDKs in TypeScript.

```

Comment:
Why the term "backend" when there is no "frontend" in this context?

Reply:
the frontend is m4 but I understand there is no need to be overly academic here :D I will replace `backend` with `extension`.